src/transformStateWithClones.js

```diff
@@ -8,6 +8,36 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const result = [];
```

Comment:
suggestion: use more descriptive names for your vars (e.g. `arrayOfStates`, `states`)